src/parse/errors.rs

```diff
@@ -506,12 +506,16 @@ impl Error {
         let mut c = Colorizer::new(true, color);
         let suffix = suggestions::did_you_mean(&bad_val, good_vals.iter()).pop();
 
-        let mut sorted: Vec<String> = good_vals.iter().map(|v| v.to_string()).collect();
+        let mut sorted: Vec<String> = good_vals
+            .iter()
+            .map(|v| v.to_string())
+            .map(|v| if v.contains(char::is_whitespace) { format!("{:?}", v) } else { v })
+            .collect();
         sorted.sort();
 
-        start_error(&mut c, "'");
-        c.warning(bad_val.clone());
-        c.none("' isn't a valid value for '");
+        start_error(&mut c, "");
+        c.warning(format!("{:?}", bad_val.to_string()));
```

Comment:
Clippy complaining here

Reply:
Sorry, `just lint` didn't remark on this one. Fixed it.